packages/web-react/config/rollup.config.js

```diff
@@ -93,7 +100,7 @@ function prepareCJSMinified(input) {
 function prepareBundle({
   dirs,
   bundleName = dirs[dirs.length - 1],
-  extensions,
+  // extensions,
```

Comment:
Let's maybe remove the line instead of commenting out?

Reply:
Removed.